k8s/pkg/apis/longhorn/v1beta1/node.go

```diff
@@ -70,6 +71,7 @@ type NodeSpec struct {
 	// +optional
 	EvictionRequested bool `json:"evictionRequested"`
 	// +optional
+	// +nullable
```

Comment:
Remove this.

Reply:
The Tags is NULL after creating the CR.
If remove the `//+ nullable`, the UpdateStatus will fail.

```
failed to update=Node.longhorn.io \“ku50-master\” is invalid: [spec.disks.default-disk-745ff2be8d312b52.tags: Invalid value: \“null\“: spec.disks.default-disk-745ff2be8d312b52.tags in body must be of type array: \“null\“, status.diskStatus: Invalid value: \“null\“: status.diskStatus in body must be of type object: \“null\“]”
```

Reply:
This is caused by `diskSpec.Tags` rather than `nodeSpec.Tags`. Can you add the initialization when adding a new disk to nodeSpec?

Reply:
Oh! Right.  
> Can you add the initialization when adding a new disk to nodeSpec?
No prob. Trying to add it.